src/main/java/com/echobox/api/linkedin/client/DefaultLinkedInClient.java

```diff
@@ -452,7 +452,7 @@ public AccessToken obtainUserAccessToken(String appId, String appSecret, String
     ValidationUtils.verifyParameterPresence("verificationCode", verificationCode);
     
     try {
-      this.webRequestor = new DefaultWebRequestor(appId, appSecret);
```

Comment:
Seems to be okay as this is done in the[ main repo](https://github.com/ebx/main/blob/dev/SocialAPI/SocialAPIApp/src/main/java/sh/ebx/api/v3/apiconnect/helper/DefaultLinkedInHelper.java#L51) without this constructor; these values are anyway passed in as parameters.

Reply:
See conversation [here](https://echoboxhq.slack.com/archives/C03U1M3BCJY/p1715093937746519) 😆